nav2_bringup/README.md

```diff
@@ -45,12 +45,25 @@ ros2 launch turtlebot3_bringup turtlebot3_state_publisher.launch.py use_sim_time
 
 ### Terminal 3: Launch Nav2
 
+normal bringup
+
 ```bash
 source /opt/ros/dashing/setup.bash
 ros2 launch nav2_bringup bringup_launch.py use_sim_time:=True autostart:=True \
 map:=<full/path/to/map.yaml>
 ```
 
+manual composed bringup
+
+```bash
+source /opt/ros/dashing/setup.bash
+ros2 launch nav2_bringup manual_composed_bringup_launch.py use_sim_time:=True autostart:=True \
+map:=<full/path/to/map.yaml>
+```
+
+* composed bringup is based on [ROS2 Composition](https://docs.ros.org/en/galactic/Tutorials/Composition.html), which is useful for embedded systems users that need to make optimizations due to harsh resource constraints.
+* some discussions about performance improvement could be found at https://discourse.ros.org/t/nav2-composition/22175
+
```

Comment:
Unrelated to this line: but you said you registered these nodes as components to do some testing? Did that work out OK? If so, please submit a new PR with the composition registration so that once we can dynamically compose the nodes, all that stuff is already ready to go.

Reply:
> you said you registered these nodes as components to do some testing?

yes,  these nodes could registered as components.

> Did that work out OK?

no, it can't work as expected, because internal node in components couldn't load parameters (e.g. `local_costmap` in `controller_server`),  you could find details here https://github.com/ros-planning/navigation2/issues/2147#issuecomment-915890304.